### PR TITLE
(test) Add tests for `add-patient-to-patient-list-menu-item.component.tsx`

### PR DESCRIPTION
**Pull request by ayush-AI**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary

In this PR I have added the tests for `add-patient-to-patient-list-menu-item.component.tsx`

## Screenshots

*None.*

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

**Comment by ayush-AI** [comment]
cc/ @Piumal1999 @jayasanka-sack @anjula-sack

File: packages/esm-patient-list-app/src/add-patient-to-patient-list-menu-item.test.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import AddPatientToPatientListMenuItem from './add-patient-to-patient-list-menu-item.component';
+import { showModal } from '@openmrs/esm-framework';
+
+const mockedShowModal = showModal as jest.Mock;
+
+jest.mock('@openmrs/esm-framework');
+
+describe('AddPatientToPatientListMenuItem', () => {
+  it('renders the button with the correct title', () => {
+    render(<AddPatientToPatientListMenuItem patientUuid="someUuid" />);
```
Comment (Piumal1999) [inline packages/esm-patient-list-app/src/add-patient-to-patient-list-menu-item.test.tsx:12]: Use a sample uuid

File: packages/esm-patient-list-app/src/add-patient-to-patient-list-menu-item.test.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    const button = screen.getByRole('menuitem');
+
+    expect(button).toBeInTheDocument();
+    expect(button).toHaveTextContent('Add to list');
+    expect(button).toHaveAttribute('title', 'Add to list');
+  });
+
+  it('should open the modal on button click', async () => {
+    render(<AddPatientToPatientListMenuItem patientUuid="someUuid" />);
```
Comment (Piumal1999) [inline packages/esm-patient-list-app/src/add-patient-to-patient-list-menu-item.test.tsx:21]: Same here

**Review by Piumal1999 — CHANGES_REQUESTED** [review]

File: packages/esm-patient-list-app/src/add-patient-to-patient-list-menu-item.test.tsx (hunk 2 above)
Comment (ayush-AI) [reply packages/esm-patient-list-app/src/add-patient-to-patient-list-menu-item.test.tsx]: Done @Piumal1999

**Review by anjula-sack — APPROVED** [review]

**Pull request merged**